.devcontainer/docker-compose.yaml

```diff
@@ -27,6 +27,9 @@ services:
       POSTGRES_USER: admin
       POSTGRES_PASSWORD: postgres
 
+    ports:
+      - "5432:5432"
+    # For local development, we need to forward the database port here too.
     # Add "forwardPorts": ["5432"] to **devcontainer.json** to forward PostgreSQL locally.
     # (Adding the "ports" property to this file will not forward from a Codespace.)
```

Comment:
Hm. I don't seem to have ports for the DB in any of my devcontainers. Where did you run into the issue? I'm curious why/when it becomes needed.

Reply:
Yesterday while running playwright tests, I couldn't run them inside the container so I ran the tests on my local env.
I kept getting connection error 127.0.0.1 port: 5432 connection not available, I did two rebuilds for the container and it didn't work until I forwarded the port.

I tried again today after removing this line and it seems to work without it too. Don't know why wasn't it working yesterday.